tests/queries/tests.py

```diff
@@ -3690,6 +3690,58 @@ def test_ticket_22429(self):
         self.assertSequenceEqual(queryset, [st2])
 
 
+class Ticket28297Tests(TestCase):
+    def test_join_use_correct_table_28297(self):
+        """
+        Query was assigning wrong table.
+        It should get the last table created by the join
+        """
+        i1 = Ingredient.objects.create(name="tomato")
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).